sphinx_automodapi/tests/test_cases.py

```diff
@@ -110,6 +117,8 @@ def test_run_full_case(tmpdir, case_dir):
             path_relative = os.path.relpath(path_reference, output_dir)
             path_actual = os.path.join(docs_dir, path_relative)
             assert os.path.exists(path_actual)
-            actual = io.open(path_actual, encoding='utf8').read()
-            reference = io.open(path_reference, encoding='utf8').read()
+            with io.open(path_actual, encoding='utf8') as f:
```

Comment:
I could be wrong here, but I think this *might* cause trouble in locales where UTF8 doesn't come out as a default?  Might it be sufficient to just do a Byte-wise comparison to sidestep the encoding problem?